pgcli/packages/pgspecial/main.py

```diff
@@ -18,6 +18,32 @@
 class CommandNotFound(Exception):
     pass
 
+
+@export
+class PGSpecial(object):
+    def __init__(self):
+        self.timing_enabled = True
+
+        self.commands = COMMANDS.copy()
+        self.register(self.show_help, '\\?', '\\?', 'Show Help.',
+                      arg_type=NO_QUERY)
+
+    def register(self, *args, **kwargs):
+        register_special_command(*args, command_dict=self.commands, **kwargs)
+
+    def execute(self, cur, sql):
+        return execute(cur, sql, self.commands)
```

Comment:
There's a weird divide in the special commands, where half of them are just shortcuts for longer queries. It makes sense to me, for pgspecial as a standalone package, to be able to run those commands without having to instantiate an an object. It's not important for pgcli though.